plugins/dbnd-docker/src/dbnd_docker/kubernetes/kubernetes_engine_config.py

```diff
@@ -92,15 +116,18 @@ class KubernetesEngineConfig(ContainerEngineConfig):
     requests = parameter.none()[Dict]
     limits = parameter.none()[Dict]
 
-    pod_exit_code_to_retry_count = parameter(empty_default=True).help(
-        "Mapping between pod exit code to amount of pod retry attempts"
-    )[Dict]
-    pod_retry_delay = parameter.help(
-        "The delay between each pod retry attempt in time delta format. 1m, 5s, 1h, etc."
+    pod_error_cfg = parameter(
+        default=PodRetryConfiguration(
+            {
+                "255": {"retry_count": 3, "retry_delay": "3m"},
+                "err_image_pull": {"retry_count": 0, "retry_delay": "3m"},
+            }
+        )
+    ).help("Pod error handling configuration.")[Dict]
```

Comment:
Should be of type `PodRetryConfiguration`, rather than `Dict`

Reply:
True